static_assets.py

```diff
@@ -19,28 +19,7 @@ def init_assets(app=None, auto_build=False, static_folder="app/static/dist"):
         # This file needs to be shipped with your code.
         env.manifest = "file"
 
-        # js = Bundle(
-        #     "./js/namespaces.js",
-        #     "./js/helpers.js",
-        #     "./js/all.js",
-        #     "./js/fsd_cookies.js",
-        #     "./js/components/**/*.js",
-        #     filters="jsmin",
-        #     output="js/main.min.js",
-        # )
-
-        # css = Bundle(
-        #     "./scss/*.scss",
-        #     filters="pyscss,cssmin",
-        #     output="css/main.min.css",
-        #     extra={"rel": "stylesheet/css"},
-        # )
-
-        # env.register("default_styles", css)
-        # env.register("main_js", js)
-
         bundles = []
```

Comment:
Does this file even ... do anything? There's no bundles, so nothing gets built? Or is there some magic here?

Reply:
Hm yeh as far as I can tell it doesn't do anything - the only thing it does as an empty file is show how it could fit into the lifecycle of the app starting + a static `build.py` process. I think there's plenty of precedent to follow for that in other apps so we could just remove it here?

Reply:
yeah, I think if it's not used at all that sounds fair 👍 

Maybe one day it will be integrated... 👀